### PR TITLE
ci: add sleep to latest release

### PR DESCRIPTION
**Pull request by Xunzhuo**

### CODE REVIEW

**Comment by zirain** [comment]
you should verify it's deleted via GH cli/api?

**Comment by zirain** [comment]
like a retry until?

**Comment by Xunzhuo** [comment]
It could, I think the issue here may be solved by easier way, just after calling gh apis, the wait sometime to make it take effect.

File: .github/workflows/latest_release.yaml [hunk 1]
```diff
@@ -29,15 +29,17 @@ jobs:
       continue-on-error: true
       run: |
         gh release delete latest --repo $GITHUB_REPOSITORY
+        sleep 3
```
Comment (arkodg) [inline .github/workflows/latest_release.yaml:32]: should we start with sleep only on L32 first ?

Comment (arkodg) [reply .github/workflows/latest_release.yaml:32]: also would be good to add a comment explaining the WHY

Comment (Xunzhuo) [reply .github/workflows/latest_release.yaml:32]: I think all sleeps matters. GH APIs take sometime to make effect, we should make sure before `Recreate the Latest Release and Tag` tag and release all get deleted.

And after `Recreate the Latest Release and Tag`, we need to sleep some time to wait new tag and release are created before checking it.

Comment (arkodg) [reply .github/workflows/latest_release.yaml:32]: Okay can you add this info as comments above each sleep ?

Comment (Xunzhuo) [reply .github/workflows/latest_release.yaml:32]: Yes.

**Review by arkodg — APPROVED** [review]
yolo

**Pull request merged**